drivers/mfd/mfd_npm6001.c

```diff
@@ -60,9 +70,46 @@ static int mfd_npm6001_init(const struct device *dev)
 		return ret;
 	}
 
+	/* If BUCK1 is not used, disable its core components to save power */
```

Comment:
I'm concerned about configuring bucks in the MFD driver, and the repetition of existing code from that driver.
Please take a look at an alternative approach in https://github.com/zephyrproject-rtos/zephyr/pull/69319

Reply:
Allright, I'll take that route. Removed the BUCK1/2 part from this PR